WebInkEnhancement/explainer.md

```diff
@@ -130,9 +132,23 @@ interface InkPresenter {
 
 interface PenStrokeTipPresenter : InkPresenter {
     void setLastRenderedPoint(PointerEvent evt, PenStrokeStyle style);
+    
+    attribute Element presentationArea;
+    readonly attribute unsigned long expectedImprovement;
 }
 ```
 
+## Interface Details
+
+Due to uncertainty around the correct execution when `setLastRenderedPoint` is called before setting the stroke style, and it being likely that the radius could change frequently, we decided it may be best to require all relevant properties of rendering the ink stroke in every call to `setLastRenderedPoint`.
+
+Instead of providing `setLastRenderedPoint` with a PointerEvent, just providing x and y values is also an option. It was decided that a trusted pointer event would likely be the better option though, as then we can have easier access to the pointer ID and the site author doesn't have to put extra thought into the position of the ink.
+
+Providing the presenter with the canvas allows the boundaries of the drawing area to be determined. This is necessary so that points and ink outside of the desired area aren't drawn when points are being forwarded. If no canvas is provided, then the containing viewport size will be used.
+
+The `expectedImprovement` attribute exists to provide site authors with information regarding the perceived latency improvements they can expect by using this API. The attribute will return the expected average number of milliseconds that latency will be improved by using the API, including prediction.

```

Comment:
nit: change 'milliseconds that latency will be improved' to 'milliseconds that perceived latency will be improved'